todo_app/trello_items.py

```diff
@@ -0,0 +1,77 @@
+import os
+import requests
+import json
+
+# Get environment variables
+TRELLOAPIKEY = os.getenv('TRELLOAPIKEY')
+TRELLOAPITOKEN = os.environ.get('TRELLOAPITOKEN')
+TODOID='6048d81adf46ed3c0e396169'
```

Comment:
We should extract these (and the board id in your urls too) into environment variables set in the `.env` file, otherwise this code can only be used against your existing trello board